bitnami/apisix/templates/_helpers.tpl

```diff
@@ -336,6 +336,16 @@ Init container definition for waiting for the database to be ready
       ln -sf /opt/bitnami/apisix/deps /usr/local/apisix
       ln -sf /opt/bitnami/apisix/openresty/luajit/share/lua/*/apisix /usr/local/apisix
       mkdir -p /usr/local/apisix/logs
+      # Clean up UDS file before starting APISIX
```

Comment:
I'm sorry I didn't notice that you had already included this in the init container.

Then replacing the container command with this shouldn't be needed.